### DISCUSSION TITLE
Getting 404 Error on https://raw.githubusercontent.com/Microsoft/CloudAdoptionFramework/master/ready/readiness-checklist.docx

### DISCUSSION THREAD

**Discussion by joykaczmarek** [community/community#88745] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Does anyone have a MS Cloud Adoption Framework Readiness Checklist?  I am getting a 404 error for this site: https://[raw.githubusercontent.com/Microsoft/CloudAdoptionFramework/master/ready/readiness-checklist.docx](https://raw.githubusercontent.com/Microsoft/CloudAdoptionFramework/master/ready/readiness-checklist.docx)

**Comment by yazan-metax** [comment]
I would like to help but no unfortunately
